k8s/mysql-deployment.yaml

```diff
@@ -96,7 +144,26 @@ spec:
         volumeMounts:
         - name: mysql-persistent-storage
           mountPath: /var/lib/mysql
+        - name: initdb-volume
+          mountPath: /scripts
+        lifecycle:
+          postStart:
+            exec:
+              command: 
+                - "/bin/sh"
+                - "-c"
+                - |
+                  while ! mysqladmin ping -uroot -p$MYSQL_ROOT_PASSWORD; do
+                    sleep 10;
+                  done
+                  mysql -uroot -p$MYSQL_ROOT_PASSWORD < /scripts/create_schema.sql
```

Comment:
Does this need a space between `-p` and the variable?

Reply:
No, there must not be a space